SPECS/mstflint/mstflint.spec

```diff
@@ -15,6 +17,8 @@ BuildRequires:  automake
 BuildRequires:  boost-devel
 BuildRequires:  gcc
 BuildRequires:  gcc-c++
+BuildRequires:  kernel-devel
+BuildRequires:  kmod
```

Comment:
Double-checking: is `kmod` needed to build the module? Or is it only needed during run-time to load it?

Reply:
Moved it to "Requires:".